unit/solvers/smt_switch.cpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************\
+
+Module: Unit tests for checking the integration of SMT switch
+
+Author: Diffblue Ltd.
+
+\*******************************************************************/
+
+#include <testing-utils/use_catch.h>
+
+// only enable the tests if compiling with SMT switch
+#ifdef HAVE_SMT_SWITCH
+#  include <cvc4_factory.h>
```

Comment:
What’s with the spaces after `#`?

Reply:
This is how my clang-format wants to format this :man_shrugging: